internal/db/invoice_test.go

```diff
@@ -0,0 +1,32 @@
+package db
+
+import (
+	"testing"
+	"time"
+
+	"github.com/slovak-egov/einvoice/internal/entity"
+	"github.com/slovak-egov/einvoice/pkg/timeutil"
+)
+
+func createTestInvoice(t *testing.T, test, isPublic bool) *entity.Invoice {
```

Comment:
hmm what about creating `internal/testutil` package so we don't repeat ourselves?
we can move there everything from `internal/apiserver/helpers_test.go`
and we can always pass connector as argument to these functions